### PR TITLE
Backup the asset dependencies during the base ami generation to save time later

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@deads2k this keeps the install-assets from taking so long, so now install-assets during the devenv ami would be much faster.  As I'm typing this I'm realizing I do need to do a copy and not a move so that test/merge which are launched from a devenv ami still have the asset deps backed up.

cc @liggitt

**Comment by jwforres** [comment]
@danmcp

File: lib/vagrant-openshift/action.rb [hunk 1]
```diff
@@ -42,13 +42,13 @@ def self.install_openshift(options)
           b.use SetHostName
           b.use InstallOpenshift
           b.use InstallOpenshiftRhel7
-          b.use InstallOpenshiftAssetDependencies
+          b.use InstallOpenshiftAssetDependencies, :backup_assets => false
```
Comment (liggitt) [inline lib/vagrant-openshift/action.rb]: :restore_assets => true

File: lib/vagrant-openshift/action/install_openshift_asset_dependencies.rb [hunk 2]
```diff
@@ -20,29 +20,52 @@ module Action
...
-  gem install bundler
-fi
+ASSET_BACKUP_DIR=/data/asset_dependencies
 
 if ! which npm > /dev/null 2>&1 ; then
   sudo yum -y install npm
 fi
 
+}
+          if !@options[:backup_assets]
```
Comment (liggitt) [inline lib/vagrant-openshift/action/install_openshift_asset_dependencies.rb]: if @options[:restore_assets]

**Pull request merged**